utilities/decrypt_data/decrypt.py

```
@@ -0,0 +1,90 @@
+import subprocess
+import psycopg2
+import json
+
+# Configuration
+CONFIG = {
+    "db": {
+        "host": "DB_hostname",
+        "port": "DB_port_number",
+        "dbname": "DB_name",
+        "user": "DB_username",
+        "password": "DB_password"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)